staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -648,1345 +826,3137 @@ testCases:
       nonMergingIntList:
         - 2
         - 3
-  - description: merge lists of scalars
-    original:
-      mergingIntList:
-        - 1
-        - 2
-    twoWay:
-      mergingIntList:
-        - 3
-    modified:
-      mergingIntList:
-        - 1
-        - 2
-        - 3
-    current:
-      mergingIntList:
-        - 1
-        - 2
-        - 4
-    threeWay:
-      mergingIntList:
-        - 3
-    result:
-      mergingIntList:
-        - 1
-        - 2
-        - 3
-        - 4
-  - description: merge lists of maps
+  - description: delete all maps from merging list
```

Comment:
There is a comment
```
// test case for setElementOrder
```
All the test cases below it are new test cases.
The others are move from regular test case to raw test cases since we need to add directive.